Source/WebKit/UIProcess/API/APIInjectedBundleClient.h

```diff
@@ -44,6 +45,9 @@ class InjectedBundleClient {
     virtual void didReceiveMessageFromInjectedBundle(WebKit::WebProcessPool&, const WTF::String&, API::Object*) { }
     virtual void didReceiveSynchronousMessageFromInjectedBundle(WebKit::WebProcessPool&, const WTF::String&, API::Object*, CompletionHandler<void(RefPtr<API::Object>)>&& completionHandler) { completionHandler(nullptr); }
     virtual RefPtr<API::Object> getInjectedBundleInitializationUserData(WebKit::WebProcessPool&) { return nullptr; }
+#if (PLATFORM(GTK) || PLATFORM(WPE))
```

Comment:
No need for these parentheses:

```suggestion
#if PLATFORM(GTK) || PLATFORM(WPE)
```

This API looks like it could be built unconditionally. Even if it won't be used on other ports, it's nicer to avoid the extra build guards when possible. We have to get the change approved by a WK2 owner regardless, so might as well ask if they'd be OK with this outside our guards.

Reply:
Hi WK2 owners, do you have a preference here?

Reply:
Not an owner, but I my feeling is that it would be nicer to not have the guards, too.